server/controllers/availability.js

```diff
@@ -0,0 +1,37 @@
+const asyncHandler = require('express-async-handler');
+const { google } = require('googleapis');
+const User = require('../models/User');
+
+
+exports.checkAvailablitiy =  asyncHandler(async(res, req, next) => {
+  const { hours } = await User.findOne({ availability: req.params.hours });
+  const calendar = 'primary';
+  const cal = google.calendar({
+    version: 'v3',
+    auth: `${process.env.GOOGLE_CLIENT_ID}`
+  });
+
+  // Make the query
+  cal.freebusy.query({
+      resource: {
+          // Set times to ISO strings as such
+          timeMin: hours[0].toISOString(), 
+          timeMax: hours[1].toISOString(),
+          timeZone: 'UTC',
+          items: [{ id: calendar }]
+      }
```

Comment:
With the routing I noticed we have a day url parameter but that is not being used. For this method, is there any way we could utilize perhaps a query parameter of todays date that the front-end passes down which will give the freebusy for that particular day?